### PR TITLE
GeckoView 88.0 RC 1

### PR DESCRIPTION
**Pull request by pocmo**
Updating the AC 74 release branch to the first GeckoView 88.0 RC build.

### CODE REVIEW

File: components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt [hunk 1]
```diff
@@ -1127,7 +1127,7 @@ class GeckoEngineSessionTest {
     }
 
     @Test
-    fun `WHEN updating tracking protection with a recommended policy THEN etpEnabled should be enabled`() {
+    fun `WHEN updateing tracking protection with a recommended policy THEN etpEnabled should be enabled`() {
```
Comment (cadeyrn) [inline components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt:1130]: typo

**Comment by st3fan** [comment]
Let's land https://github.com/mozilla-mobile/android-components/pull/10079 before this one.

File: components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt (hunk 1 above)
Comment (pocmo) [reply components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt:1130]: Oh. This code was just moved over. This means this typo must be in the beta and probably nightly component too.

**Review by jonalmeida — APPROVED** [review]

File: components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt (hunk 1 above)
Comment (st3fan) [reply components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt:1130]: Not blocking.

**Pull request merged**